spring-data-neo4j/src/main/java/org/springframework/data/neo4j/repository/query/AbstractGraphRepositoryQuery.java

```diff
@@ -28,12 +29,14 @@
  */
 public abstract class AbstractGraphRepositoryQuery implements RepositoryQuery {
 
-	private final GraphQueryMethod method;
-	private final Session session;
+	protected final GraphQueryMethod queryMethod;
+	protected final MetaData metaData;
+	protected final Session session;
 
-	protected AbstractGraphRepositoryQuery(GraphQueryMethod method, Session session) {
+	protected AbstractGraphRepositoryQuery(GraphQueryMethod queryMethod, MetaData metaData, Session session) {
```

Comment:
You might want to keep polishing in a separate commit making the actual change more obvious.